build.gradle

```diff
@@ -1179,6 +1190,14 @@ project('test:system') {
         if (System.getProperty("imageVersion") == null) {
             systemProperty "imageVersion", "INVALID_IMAGE_VERSION"
         }
+        // validating that bookkeeper operator version >= 0.1.3
+        if (System.getProperty("bookkeeperOperatorVersion") ==~ /^(0)\.(1)\.(0|1|2)(-.+)*/) {
+            throw new GradleException('Requires Bookkeeper Operator Version >= 0.1.3')
```

Comment:
Perhaps we can use InvalidUserDataException to convey this.
https://docs.gradle.org/current/javadoc/org/gradle/api/InvalidUserDataException.html

Reply:
Done